.github/workflows/browser-tests.js.yml

```diff
@@ -0,0 +1,18 @@
+name: Browser tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+
+jobs:
+  browser-tests:
+    runs-on: ubuntu-latest
```

Comment:
Question: should we pin the ubuntu version here, so we don't get surprise updates at some point in the future?

Reply:
which version you suggested? Or I just use the current one because it works?

Reply:
It seems `24.04` is the latest one available https://github.com/actions/runner-images